bulkExport/glueScripts/export-script.py

```diff
@@ -141,7 +141,7 @@ def is_included_in_group_export(resource, group_member_ids, group_patient_ids, c
     compartment_search_params = json.load(s3Obj['Body'])
 
     print('Extract group member ids')
-    group_members = Filter.apply(frame = filtered_tenant_id_frame, f = lambda x: x['id'] == group_id).toDF().collect()[0]['member']
+    group_members = Filter.apply(frame = filtered_tenant_id_frame, f = lambda x: x['id'] == group_id).toDF().collect()[-1]['member']
```

Comment:
Can we add a sort here like`group_members = Filter.apply(frame = filtered_tenant_id_frame, f = lambda x: x['id'] == group_id).toDF().sort("meta.versionId").collect()[-1]['member']` ? I can also take this and create a separate PR if that's easier.